jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SlideGroupsTests.java

```diff
@@ -1,26 +1,22 @@
 package io.github.epam.vuetify.tests.complex;
 
-import com.epam.jdi.light.vuetify.elements.complex.SlideGroup;
 import io.github.epam.TestsInit;
-import io.github.epam.vuetify.tests.data.SlideGroupTestsData;
+import org.openqa.selenium.By;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-import java.util.List;
-
 import static com.epam.jdi.light.elements.common.WindowsManager.resizeWindow;
 import static com.jdiai.tools.Timer.waitCondition;
 import static io.github.com.StaticSite.slideGroupsPage;
 import static io.github.com.pages.SlideGroupsPage.activeClassSlideGroup;
 import static io.github.com.pages.SlideGroupsPage.centerActiveSlideGroup;
-import static io.github.com.pages.SlideGroupsPage.customIconsSlideGroup;
 import static io.github.com.pages.SlideGroupsPage.multipleSlideGroup;
 import static io.github.com.pages.SlideGroupsPage.pseudoCarouselSlideGroup;
+import static io.github.com.pages.SlideGroupsPage.pseudoCarouselCount;
+import static org.testng.Assert.assertEquals;
 
 public class SlideGroupsTests extends TestsInit {
 
-    SlideGroupTestsData testsData = new SlideGroupTestsData();
-
     @BeforeClass
     public void before() {
```

Comment:
Use show(): at the beginning of every test. it is visible on a screenshot in case of error